### ISSUE TITLE
Can not boot Talos 1.8.*

### ISSUE THREAD

**Issue by Blarc** [siderolabs/talos#9529]
### Description

Talos 1.8.1 does not boot from USB. I also tried booting Talos 1.7.0 and then applying machine configuration with Talos 1.8.1 specified and I get the same error. I choose the reset option when booting multiple times, but that doesn't help.

Could it be that my disk is somehow corrupted from previous installs on this PC? I thought that the `Reset Talos installation...` wipes the disk though.

Talos 1.7.7 installs fine.

### Logs
It's hard to copy logs from bootloader process, but these are the lines before the shutdown:
```
talos error running phase 6 when initialize sequence
failed to revert bootloader unexpected EOF
```

### Environment

- Talos version: 1.8.1
- Kubernetes version: 1.30.1
- Platform: bare-metal

**Comment by smira** [comment]
It's not clear what the issue is without any logs.

The error you posted looks like an attempt to revert an upgrade which is strange.

**Comment by MichelJansen91** [comment]
I have the same issue. It seems the device gets stuck in an infinite boot loop.
I can't get the logs during startup as `talosctl dmesg` doesnt yet work. 

Any suggestions on how to get the error message where in the process it fails?

Happened both when upgrading from 1.7.x and clean 1.8.* install.

**Comment by smira** [comment]
I wonder if it's https://github.com/siderolabs/talos/issues/9369

**Comment by Blarc** [comment]
I'll try to record the boot process.

**Comment by MichelJansen91** [comment]
Adding `i915` and `intel-mei` drivers to the talos image fixed the issue for me indeed. Thanks @smira

**Comment by Blarc** [comment]
> Adding `i915` and `intel-mei` drivers to the talos image fixed the issue for me indeed. Thanks @smira

This worked for me as well! Thanks a lot :pray: .

Closing as duplicate of https://github.com/siderolabs/talos/issues/9369.

**Comment by DizzieNight** [comment]
I am having the same issue now. I added i915 and intel-mei extensions to the iso but it still won't work. Just before it reboots I get a `WARNING: Failed to Drain Controllers` as well as the error above.

I did boot it successfully the first time but talos didn't wipe the drive it was installing to. So I booted up a live fedora iso and formatted the internal drive (which had batocera on it from a previous owner) and now whenever I try to install talos I get the `failed to revert bootloader unexpected EOF` error.
